documentation/book/proc-downgrading-brokers-older-kafka.adoc

```diff
@@ -0,0 +1,87 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrading-kafka-versions.adoc
+
+
+[id='proc-downgrading-brokers-older-kafka-{context}']
+
+= Downgrading brokers to an older Kafka version
+
+You can downgrade a {ProductName} Kafka cluster from one version to a lower version; for example, from 2.1.0 to 2.0.0.
+
+IMPORTANT: Downgrading is not possible if the version being downgraded _from_ has ever used a `log.message.format.version` that's not supported by the version being downgraded _to_ (including where the default value for `log.message.format.version` is used). 
```

Comment:
```suggestion
IMPORTANT: Downgrading is not possible if the version being downgraded _from_ has ever used a `log.message.format.version` that is not supported by the version being downgraded _to_ (including where the default value for `log.message.format.version` is used). 
```

Reply:
Later on we start talking about the previous and new versions. There's potential for confusion there since this is downgrade, so is that new as in the version being downgraded to, or new as in the newer version which we're downgrading from. I wonder if we should define terms here in the intro to avoid potential for confusion. Wdyt?